packages/transaction-controller/src/TransactionController.ts

```diff
@@ -2008,53 +1981,6 @@ export class TransactionController extends BaseController<
     }
   }
 
-  /**
-   * Append new send flow history to a transaction.
-   *
-   * @param transactionID - The ID of the transaction to update.
-   * @param currentSendFlowHistoryLength - The length of the current sendFlowHistory array.
-   * @param sendFlowHistoryToAdd - The sendFlowHistory entries to add.
-   * @returns The updated transactionMeta.
-   */
-  updateTransactionSendFlowHistory(
```

Comment:
If we're avoiding breaking changes in this PR, should we empty this and mark it deprecated?

Reply:
Marked as deprecated and removed code.